### PR TITLE
Add NLog

### PR DESCRIPTION
**Pull request by zhamppx97**
I added logging NLog, could you please review my PR

### CODE REVIEW

File: content/programming-cookbook/logging/NLog.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: NLog
+showMetadata: true
+editable: true
+showToc: true
+order: 3
```
Comment (aaronamm) [inline content/programming-cookbook/logging/NLog.md]: I made a mistake by adding unnecessary order: 3 in `log4net.md`, could you please remove it. We don't need any order in logging content md file.

File: content/programming-cookbook/logging/NLog.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- Create a new .NET Framework console project. This can be applied to .NET Core
+- Install NLog Nuget package.
+```
+install-package NLog
+
+```
+
+- Edit NLog.config file
+```
+    <?xml version="1.0" encoding="utf-8" ?>
```
Comment (aaronamm) [inline content/programming-cookbook/logging/NLog.md]: Very minor, I think for XML we can make 2 space indention, could you please update this XML code to use 2 indention.

File: content/programming-cookbook/logging/NLog.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+- Create a new .NET Framework console project. This can be applied to .NET Core
+- Install NLog Nuget package.
+```
+install-package NLog
+
+```
+
+- Edit NLog.config file
+```
```
Comment (aaronamm) [inline content/programming-cookbook/logging/NLog.md]: Very minor, you can add xml after ``` for a better syntax highlight.

File: content/programming-cookbook/logging/NLog.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+     <rules>
+        <logger name="Logfile" minlevel="Trace" writeTo="Logfile" />
+     </rules>
+    </nlog>
+
+```
+
+- Set up the main class.
+```
```
Comment (aaronamm) [inline content/programming-cookbook/logging/NLog.md]: Very minor, you can add c# after ``` for a better syntax highlight.

**Comment by aaronamm** [comment]
@zhamppx97 FYI, I have already fixed log4net.md. You use it as an example.
https://raw.githubusercontent.com/dotnetthailand/dotnetthailand.github.io/main/content/programming-cookbook/logging/log4net.md
Thanks.

**Comment by zhamppx97** [comment]
@aaronamm All done, Thanks for your comment 😀

**Comment by aaronamm** [comment]
@zhamppx97 Thanks for your contribution.

**Comment by aaronamm** [comment]
https://www.dotnetthailand.com/credit
Thanks

**Pull request merged**